src/nomad_simulations/schema_packages/properties/thermodynamics.py

```diff
@@ -0,0 +1,289 @@
+        super().normalize(archive, logger)
+
+
+class ChemicalPotential(BaseEnergy):
+    """
+    Free energy cost of adding or extracting a particle from a thermodynamic system.
+    """
+
+    # ! implement `iri` and `rank` as part of `m_def = Section()`
+
+    iri = 'http://fairmat-nfdi.eu/taxonomy/ChemicalPotential'
+
+    def __init__(
+        self, m_def: 'Section' = None, m_context: 'Context' = None, **kwargs
+    ) -> None:
+        super().__init__(m_def, m_context, **kwargs)
+        self.rank = []
+        self.name = self.m_def.name
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class HeatCapacityCV(PhysicalProperty):
+    """
+    Physical property section describing the heat capacity at constant volume for a (sub)system.
+    """
+
+    value = Quantity(
+        type=np.float64,
+        unit='joule / kelvin',
+        description="""
+        The value of the heat capacity.
+        """,
+    )
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class HeatCapacityCP(PhysicalProperty):
+    """
+    Physical property section describing the heat capacity at constant volume for a (sub)system.
+    """
+
+    value = Quantity(
+        type=np.float64,
+        unit='joule / kelvin',
+        description="""
+        The value of the heat capacity.
+        """,
+    )
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+################################
+# other thermodynamic properties
+################################
+
+
+class Virial(BaseEnergy):
+    """
+    Physical property section describing the virial of a (sub)system.
+    """
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class Density(PhysicalProperty):
+    """
+    Physical property section describing the density of a (sub)system.
+    """
+
+    value = Quantity(
+        type=np.float64,
+        unit='kg / m ** 3',
+        description="""
+        The value of the density.
+        """,
+    )
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+# ? I think this may fit better elsewhere
+class Hessian(PhysicalProperty):
+    """
+    Physical property section describing the Hessian matrix, i.e., 2nd derivatives with respect to geometric (typically particle) displacements,
+    of the potential energy of a (sub)system.
+    """
+
+    value = Quantity(
+        type=np.float64,
+        unit='joule / m ** 2',
+        description="""
+        The value of the Hessian.
+        """,
+    )
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+    # ? Is this ever used?
```

Comment:
I would delete this. If this is referring to the free energy of a phononic system, the PhysicalProperty will be anyways `FreeEnergy`, not this quantity